### DISCUSSION TITLE
Tasks will not open in Conductor UI v3.11.3 in Chrome or Firefox, but Edge works fine

### DISCUSSION THREAD

**Discussion by arketec** [Netflix/conductor#3442] [Q&A] [answered]
I am having a strange issue where Conductor UI does not open tasks when clicked in the executions, but this only happens in Firefox and Chrome. In Edge, the tasks open fine.

I am referring to this:
![image](https://user-images.githubusercontent.com/1336974/212173392-32c5b169-c33e-4bb3-8cbb-f5e7bdc04a66.png)

This only works in Edge, not in Firefox or Chrome:
![image](https://user-images.githubusercontent.com/1336974/212173546-2821703f-5e41-4c53-b47b-f6df693119b0.png)

I do not have any extensions or addons installed in Chrome (I do in Firefox), and I reinstalled Chrome from scratch and still have this issue

here is what I see in the browser console for Firefox and Chrome:
![image](https://user-images.githubusercontent.com/1336974/212172549-20186711-0120-466f-8a34-888eb61d5362.png)

**Comment by nhandt2021** [comment, marked as answer]
Hi @arketec,
I don't get the issue with Chrome (I'm using `Version 108.0.5359.124`)
I've fixed this issue in this PR [#3444]

**Reply by syw1995** [reply]
it doesn't work for me, latest version chrome and safari still can't work
<img width="1021" alt="image" src="https://user-images.githubusercontent.com/35061282/212643203-12bf5527-8039-4d1b-9df0-2dadf96531ec.png">

<img width="722" alt="image" src="https://user-images.githubusercontent.com/35061282/212643034-2095913f-c4a0-4c3f-8844-c406a435bac0.png">

**Comment by arketec** [comment]
Thanks for that fix

I have the latest version of Chrome as of yesterday (109.0.5414.75) and it was still not working
![image](https://user-images.githubusercontent.com/1336974/212435475-6ff72742-f8f7-4254-820a-b6b14c33e28b.png)


Update:
I just tried downgrading to an earlier version (104.0.5112.102 was what I could find online) and chrome works again, so something in the lastest brower is breaking the UI it would seem

**Comment by arketec** [comment]
@nhandt2021 
Just tested your fix in the latest Chrome version, and it will also work for that

**Comment by syw1995** [comment]
maybe this can help you
https://stackoverflow.com/questions/36845515/mouseevent-path-equivalent-in-firefox-safari

The problem was caused by the change of MouseEvent.
I  change the code "const taskRef = e.path[1].id || e.path[2].id;"  -> "const taskRef = e.composedPath()[1].id || e.composedPath()[2].id;".
It works in chrome and safari.

**Comment by sourabh30** [comment]
Hi @arketec, I have fixed the code for Chrome and Firefox in the PR https://github.com/Netflix/conductor/pull/3447

**Comment by arketec** [comment]
Using a combination of the fixes here (thanks everyone) I was able to fix our conductor as well. Still, these should be merged into main for future releases.
